src/verbs.rs

```diff
@@ -1365,3 +1365,7 @@ pub unsafe fn rdma_get_recv_comp(id: *mut rdma_cm_id, wc: *mut ibv_wc) -> c_int
         ret
     }
 }
+
+pub const IBV_LINK_LAYER_UNSPECIFIED: u8 = 0;
```

Comment:
Will bindgen generate bindings for anonymous enum automatically? I found a related MR at https://github.com/rust-lang/rust-bindgen/pull/103

Reply:
I've searched, but couldn't find a configuration to control that, and the MR mentioned above require the anonymous enum in a `class` or `struct`, so I just bind them manually.

Reply:
Got it. I tried bindgen myself and found that the issue may be related to allowlist; If I don't specify any allowlist then the bindgen will generate bindings for unnamed enums. I guess binding them manually is the only solution now.